sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_shared/async_challenge_auth_policy.py

```diff
@@ -21,13 +24,16 @@ class AsyncChallengeAuthPolicy(ChallengeAuthPolicyBase, AsyncHTTPPolicy):
             )
             if request.http_request.body:
                 # no_body was created with request's headers -> if request has a body, no_body's content-length is wrong
+                _LOGGER.debug("no_body was created with request's headers."
+                              "If request has a body, no_body's content length is wrong")
```

Comment:
This message is confusing, and I'm not sure the log entry is needed

Reply:
I see that, getting rid of it in both files